cmd/kube-apiserver/app/options/options_test.go

```diff
@@ -121,6 +122,7 @@ func TestAddFlags(t *testing.T) {
 		"--storage-backend=etcd3",
 		"--service-cluster-ip-range=192.168.128.0/17",
 		"--lease-reuse-duration-seconds=100",
+		"--emulated-k8s-version=v1.31.1-alpha",
```

Comment:
nit: I would store the version in a const and pass it here using `fmt.Sprintf("--emulated-k8s-version=v%s", version)`, that way you can use that reference in your test assertions later.